src/RepairManager/rule-config.yaml

```diff
@@ -0,0 +1,11 @@
+---  
+# [Rule Module Name] : [Rule Class Name]
+rules:
+  Rules.ecc_rule : ECCRule
+  
+# time to sleep between rule execution
+wait_time: 10
+
+# prometheus ecc error query
+ecc_prometheus_url: 'http://localhost:9091/prometheus/api/v1/query?query=nvidiasmi_ecc_error_count%7Btype%3D%22volatile_double%22%7D%3E0'
```

Comment:
Should `ecc_prometheus_url` be grouped under `ecc_rule`?

Reply:
it should be prometheus_url in config file (e.g. http://localhost:9091/prometheus or even http://localhost:9091, @xudifsd to confirm). the whole url should be assembled in ecc_rule function.

Reply:
Yes, prometheus will listen on 9091 port